src/components/taskDetails/index.tsx

```diff
@@ -0,0 +1,267 @@
+import React, {
+    ChangeEvent,
+    FC,
+    useEffect,
+    useContext,
+    useReducer,
+    useRef,
+    useState,
+    ChangeEventHandler,
+} from 'react';
+import useFetch from '@/hooks/useFetch';
+import { isUserAuthorizedContext } from '@/context/isUserAuthorized';
+import NavBar from '@/components/navBar/index';
+import { toast, ToastTypes } from '@/helperFunctions/toast';
```

Comment:
can we sort the imports here somewhat like in the format where
the library imports are first
then components
then helper functions then, CSS followed by assets

Reply:
Mahima I think, this needs to be configured so it happens automatically in each component.

Reply:
> can we sort the imports here somewhat like in the format where the library imports are first then components then helper functions then, CSS followed by assets

Done ✅